### PR TITLE
Add a README note about rendering configs before running docker compose.

### PR DESCRIPTION
**Pull request by wchamber**
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've followed [the instructions](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [x] Get two thumbsworth of review and PO signoff if necessary. Documentation PRs only need 1 thumb.
- [x] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,9 @@ You can also check [http://0.0.0.0:8432](http://0.0.0.0:8432) which should show
 
 ## Run in a Docker container
 
+First, render configs (see above). If you try to run the docker compose before rendering, subsequent rendering may not
+work. Try removing created directories and re-rendering.
```
Comment (gpolumbo-broad) [inline README.md]: Do you have an example of what the `created directories` might be and how did they get created and why can/should they be deleted?

**Review by rsasch — CHANGES_REQUESTED** [review]
I think it might be clearest if you number the steps under "Running locally" so that "Render configs" is 1, "Run on your local environment" is 2a, and "Run in a Docker container" is 2b (or something similar.   -- the two docker options will need to change to I and II or something, but you get the idea....)

File: README.md (hunk 1 above)
Comment (wchamber) [reply README.md]: added some more info, ptal

**Comment by wchamber** [comment]
> I think it might be clearest if you number the steps under "Running locally" so that "Render configs" is 1, "Run on your local environment" is 2a, and "Run in a Docker container" is 2b (or something similar. -- the two docker options will need to change to I and II or something, but you get the idea....)

Thanks! I added it as step 1 for both options and left the explanation above. That seemed the simplest with markdown auto rendering numbers.

**Review by rsasch — APPROVED** [review]
LGTM 👍

**Review by gpolumbo-broad — APPROVED** [review]

**Pull request merged**